src/App.js

```diff
@@ -3,14 +3,30 @@ import Header from './components/Header';
 import './App.css';
 import Footer from './components/Footer';
 import Card from './components/Card';
+import {useState, useEffect} from 'react';
+
 
 
 function App() {
+  const [characters, setCharacters] = useState([])
+
+const url= 'https://rickandmortyapi.com/api/character'
+
+function doFetch(){
+    fetch(url)
+    .then((response) => response.json())
+    .then((data) => setCharacters(data.results))
+}
+console.log(characters)
```

Comment:
You should remove the console.log as every finished feature should be free from console.logs. :)